packages/web-app-files/tests/unit/views/SharedViaLink.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('SharedViaLink view', () => {
+  it.todo('should call "loadResources" method on created')
+  it.todo('should call "adjustTableHeaderPosition" method on mounted')
+
+  describe('when loading is set as true', () => {
+    it.todo('should show list-loader component')
+  })
+
+  describe('when loading is set as false', () => {
+    it.todo('should not show list-loader component')
+
+    describe('when the length of activeFiles is zero', () => {
+      it.todo('should show no-content-message component')
+      it.todo('should not show oc-table-files component')
+    })
+
+    describe('when the length of activeFiles is one or more', () => {
+      it.todo('should not show no-content-message component')
+      it.todo('should show oc-table-files component with props')
+      it.todo('should set props on context-actions component')
+      it.todo('should set props on list-info component')
+      it.todo('should call "$_mountSideBar_showDefaultPanel" method on "showDetails" event')
+      it.todo('should call "$_fileActions_triggerDefaultAction" method on "fileClick" event')
+      it.todo('should call "rowMounted" method when the table row is mounted')
```

Comment:
isn't this method is called when some event like you mentioned in above step is fired? rather saying `when the table row is mounted` you can be more detailed with the event name